### ISSUE TITLE
Relative size should not be converted to absolute size

### ISSUE THREAD

**Issue by tedil** [lanoxx/tilda#279]
When a relative height/width is given (e.g. 75% height), it is automatically converted to an absolute value in pixels, (e.g. 75% of 2160px are 1620px) which is in turn stored in the config/preferences. When changing resolutions, this absolute value is used instead of the relative one (i.e. if you lower vertical resolution from say 2160px to 1080px, tilda's main window will still be 1620px tall, when it would make more sense for it to have a height of 0.75*1080px = 810px instead).

**tl;dr**: If the user does not explicitly specify an absolute height/width, always use relative size (i.e. recalculate upon change of display resolution) instead.

This is probably slightly related to #225 (i.e. fullscreen size overrides non-fullscreen size so the latter does not get restored when toggling fullscreen mode on/off [I guess]).

**Comment by kadrach** [comment]
This would be great to have, at the moment I need to reconfigure tilda when using different monitors that have different resolutions (e.g. remote display or undocking and using the builtin, etc).

**Comment by sealj553** [comment]
This is a problem for me as well when switching monitors.

**Comment by demlak** [comment]
this drives me crazy on my setup. (un-)docking laptop changes used resolution of OS. this should also affect tilda.

**Comment by TheStrangeKat** [comment]
+1 I'd be super happy if this were resolved. It's an inconvenience at worst, but switching between my work monitor and my laptop resolutions is a bit of a pain.

**Comment by lanoxx** [comment]
This has been resolved. Starting from 1.5 tilda will store the relative size in the config file. Tilda now also automatically resizes when the monitor resolution changes.

**Comment by lanoxx** [comment]
Fixed by 2eba9f584d18949fb8ba0c1553e1d2b8f274cc82 and a9b0c061cf1bca88034452b838a95bc71d457506.
